ui/components/app/snaps/snap-privacy-warning/index.scss

```diff
@@ -3,13 +3,25 @@
     max-height: 325px;
     overflow-y: auto;
 
+    &__terms-link {
+      outline: none !important;
+
+      &:focus {
+        outline: none !important;
+      }
+
+      &:focus-visible {
+        outline: none !important;
+      }
+    }
+
     &__scroll-button {
       position: absolute;
       margin-left: auto;
       margin-right: auto;
       right: 0;
       left: 0;
-      bottom: 110px;
```

Comment:
The scroll button seems very close to the bottom now. Is that intentional?

Reply:
It was in the middle of the modal if I recall. I added this to bring it down.

Reply:
That makes sense - My comment is just that if you compare the privacy notice to the other screens that have the arrow, the arrow is slightly above the bottom on every other screen.

Reply:
I had consultations with Erik about this now, and as he stated: `Should be 16px from / above the footer container`.
I checked the exact measures of the current situation with components and it is matching the expectations. Here is the proof:
![Screenshot 2024-03-06 at 11 42 03](https://github.com/MetaMask/metamask-extension/assets/13301024/267ffd39-b185-4f75-9ee4-1a12eff93796)

Reply:
The footer has its own padding of 16px so that's enough and matches Erik's request. Before we had bottom margin of 110px to make it look correct. Now that components are structured in a better way and have their own paddings/margins, I think it's easier to respect that.